python/timeplus/env.py

```diff
@@ -31,12 +36,17 @@ def workspace(self, name):
         Returns:
         Environment: The current Environment instance.
         """
-        self._workspace = name
-        if not self._address:
-            self._configuration.host = f"https//us.timeplus.cloud/{self._workspace}/api"
+        if DEV_ADDRESS in self._address:
+            self._workspace = name
+            self._configuration.host = f"{self._address}/{self._workspace}/api"
+            return self
         else:
+            if len(name) != WORKSPACE_LEN:
```

Comment:
I think we can remove this workspace validation, even for production, we actually have a `tp-monitoring` workspace that does not follow the lenth rule